conda/recipes/cuvs/recipe.yaml

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: '${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}'
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
```

Comment:
Is `py_buildstring` going to be treated as an integer instead of a string? Should it be single-quoted? (Probably applies to many recipes.)

Reply:
No, `version_to_buildstring` makes sure it is a string. I'm waiting for upstream changes to make that easier for other context variables